client/src/utils/sha-1.d.ts

```diff
@@ -0,0 +1 @@
+export declare module 'sha-1';
```

Comment:
I don't think this is needed any more.

Reply:
Removed and TS server started crying, I will give it another try.

Reply:
I'll take a look.